src/EA.Weee.Web/ViewModels/Shared/Mapping/ViewEvidenceNoteViewModelMap.cs

```diff
@@ -81,7 +81,7 @@ public ViewEvidenceNoteViewModel Map(ViewEvidenceNoteMapTransfer source)
                 AatfApprovalNumber = source.EvidenceNoteData.AatfData.ApprovalNumber,
                 DisplayEditButton = (source.EvidenceNoteData.Status == NoteStatus.Draft || source.EvidenceNoteData.Status == NoteStatus.Returned) && source.EvidenceNoteData.AatfData.CanCreateEditEvidence,
                 RedirectTab = source.RedirectTab,
-                EvidenceNoteHistoryData = mapper.Map<IList<EvidenceNoteHistoryViewModel>>(source.EvidenceNoteData.EvidenceNoteHistoryData),
+                EvidenceNoteHistoryData = mapper.Map<IList<EvidenceNoteHistoryViewModel>>(source.EvidenceNoteData.EvidenceNoteHistoryData)
```

Comment:
Can be removed a transfer note wont have the history its the original evidence note that has the history. George to update the wireframe.

Reply:
are you sure about this one ?

Reply:
Ah good spot. It looked like a new line was added but just a comma removed, ignore me. on this one.

Reply:
done